gui/src/main/java/com/devonfw/ide/gui/MainController.java

```diff
@@ -113,21 +119,19 @@ private void setWorkspaceValue() {
             .map(Path::toString)
             .forEach(name -> selectedWorkspace.getItems().add(name));
 
-        selectedWorkspace.setValue("main");
       } catch (IOException e) {
-        e.printStackTrace();
+        throw new RuntimeException("Error occurred while fetching workspace names." + e);
```

Comment:
https://github.com/devonfw/IDEasy/blob/main/documentation/coding-conventions.adoc#catching-and-handling-exceptions
```suggestion
        throw new RuntimeException("Error occurred while fetching workspace names.", e);
```